diff --git a/setup.py b/setup.py
index 84edb2d..5197580 100755
--- a/setup.py
+++ b/setup.py
@@ -257,6 +257,7 @@ if __name__ == "__main__":
         'po_package': "quodlibet",
         'shortcuts': ["quodlibet.desktop", "exfalso.desktop"],
         'man_pages': ["man/quodlibet.1", "man/exfalso.1"],
+	'libraries = " -lX11"'
         'gobject_modules': [
                     GObjectExtension("quodlibet._mmkeys",
                             "gobject_ext/mmkeys/mmkeys.defs",
